src/workerd/jsg/jsg-test.c++

```diff
@@ -115,8 +115,6 @@ KJ_TEST("inheritance") {
   e.expectEval("newExtendedAsBase(123, 'foo') instanceof NumberBox", "boolean", "true");
 
   e.expectEval("newExtendedAsBase(123, 'foo') instanceof ExtendedNumberBox", "boolean", "true");
-
-  e.expectEval("(new Other())[Symbol.toStringTag]", "string", "Other");
```

Comment:
I couldn't find this test in the PR elsewhere. Can we add this test, so we dont' lose coverage?

Reply:
It's tested in the global-scope-test